_rules/6cbcbe.md

```diff
@@ -0,0 +1,279 @@
+```
+
+#### Passed example 3
+
+Resources are not the same, since the links resolve to different URLs, but the resources are completely identical, thus serving the same purpose:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index-copy.html">Contact us</a>
+```
+
+#### Passed example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a>
+<a href="/test-assets/6cbcbe/careers/contact.html">Contact us</a>
+```
+
+#### Passed example 5
+
+URLs differ due to trailing slashes, but resolves to the same resource after redirects caused by user agent:
+
+```html
+<a href="/test-assets/6cbcbe/">Contact us</a> 
+<a href="/test-assets/6cbcbe">Contact us</a>
+```
+
+#### Passed example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfils the same purpose in relation to the link:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Call us</a>
+<a href="/test-assets/6cbcbe/page2.html">Call us</a>
+```
+
+#### Passed example 7
+
+Pages have the same advertised key content but use different layouts:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Contact us</a>
+<a href="/test-assets/6cbcbe/page3.html">Contact us</a>
+```
+
+#### Passed example 8
+
+Links created via scripting with explicit role of link, but leads to the same resource: 
+
+```html
+<span role="link" tabindex="0"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+
+<span role="link" tabindex="0"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+```
+
+#### Passed example 9
+
+A set of two SVG `<a>` elements have the same accessible name and link to the same resource. 
+
+```html
+<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <a href="http://facebook.com" aria-label="Follow us">
+    <circle cx="50" cy="40" r="35"/>
+  </a>
+
+  <a href="http://facebook.com">
+    <text x="50" y="90" text-anchor="middle">
+      Follow us
+    </text>
+  </a>
+</svg>
+```
+
+### Failed
+
+#### Failed example 1
+
+Same link text used for links going to different resources:
+
+```html
+<a href="http://facebook.com">Follow us</a> 
+<a href="http://twitter.com">Follow us</a>
+```
+
+#### Failed example 2
+
+Same link text used for links going to web pages that are similar, but have different information in their content:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a> 
+<a href="/test-assets/6cbcbe/admissions/contact.html">Contact us</a>
+```
+
+#### Failed example 3
+
+Links created via scripting with explicit role of link, but leads to different resources that offer different content: 
```

Comment:
leads > lead

Additionally, if you drop the first "different" it might improve readability